pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -216,13 +216,14 @@ in {
   # to be adapted
   zfsStable = common {
     # check the release notes for compatible kernels
-    kernelCompatible = kernel.kernelOlder "5.19";
-    latestCompatibleLinuxPackages = linuxPackages_5_15;
+    kernelCompatible = kernel.kernelOlder "5.20";
+    latestCompatibleLinuxPackages = linuxPackages_5_19;
 
     # this package should point to the latest release.
-    version = "2.1.5";
+    version = "2.1.6";
+    rev = "6a6bd493988c75331deab06e5352a9bed035a87d";
```

Comment:
Why do you need this? There's a 2.1.6 tag: https://github.com/openzfs/zfs/releases/tag/zfs-2.1.6

Reply:
It's the same revision. I'm not sure on the protocol here, more I just want to get the ball rolling on support.

Reply:
But it looks like I can just skip the revision in this case